.github/workflows/sanity-test.yaml

```diff
@@ -97,15 +98,32 @@ jobs:
             exit 1
           fi
 
+      # generate new entries to roll into the next load test
+      # only runs if the test was successful
+      - name: New Data Creation
+        working-directory: ./src/cmd/factory
+        env:
+          AZURE_CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          AZURE_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+          AZURE_TENANT_ID: ${{ secrets.TENANT_ID }}
+          CORSO_M365_LOAD_TEST_USER_ID: ${{ secrets.CORSO_M365_LOAD_TEST_USER_ID }}
+        run: |
+          go run . exchange emails \
+          --user ${{ env.CORSO_M365_TEST_USER_ID }} \
+          --tenant ${{ env.AZURE_TENANT_ID }} \
+          --destination lt_${{ steps.repo-init.outputs.result }} \
```

Comment:
The `lt_` prefix should be reserved for load testing.  Can this get a different prefix, so we don' t have any issues cleaning it up?

Reply:
As the discussion on slack channel, I have named the folders as - `Corso_Restore_st_<current_time>`